packages/adblocker/src/engine/engine.ts

```diff
@@ -80,6 +81,12 @@ export interface BlockingResponse {
         contentType: string;
         dataUrl: string;
       };
+  urlRewrite:
```

Comment:
I struggle with this interface. On one hand, `removeparam` is just a redirect, but the `redirect` type here is very much coupled with `Resources` and extending it would be a breaking change. 

Additionally, I'm not sure if we need to list `removeparam` filters with exceptions - we don't do it for redirects. Event emitter can provide information about matched filters.